src/main/java/com/google/devtools/build/lib/rules/python/PyRuntimeInfo.java

```diff
@@ -62,6 +63,7 @@ public final class PyRuntimeInfo implements Info, PyRuntimeInfoApi<Artifact> {
   private final PythonVersion pythonVersion;
 
   private final String stubShebang;
+  @Nullable private final Artifact bootstrapTemplate;
```

Comment:
Can we remove the nullable annotations for this, or is something making it necessary? In practice, it's not actually nullable

Reply:
It is nullable since I can't make it be a FileApi.class (it complains for some reason about it being a Target and not a FileApi) so people can put anything and if it's not a Target or a SourceArtifact, then it's null

Reply:
Another thing occurred to me: PyRuntimeInfo() can be directly called by users, right? But we don't have a way to provide a default Artifact for such invocations. So yeah, it'll have to be nullable and consumers will have to handle thatcase (like how py_binary is)